doc/man/hirte-agent.1.md

```diff
@@ -30,6 +30,12 @@ The host used by `hirte-agent` to connect to `hirte`. Must be a valid IPv4 or IP
 
 The port on which `hirte` is listening for connection request and the `hirte-agent` is connecting to. This option will overwrite the port defined in the configuration file.
 
+#### **--address**, **-a**
+
+DBus address used by `hirte-agent` to connect to `hirte`. See `man sd_bus_set_address` for its format.
+Overrides any setting of `ManagerHost` or `ManagerPort`.
+This option will overwrite the port defined in the configuration file.
```

Comment:
```suggestion
Overrides any setting of `ManagerHost` or `ManagerPort` defined in the configuration file.
```

Reply:
Done. I've changed it in both files (since the address takes precedence over any host:port configuration)